WORKSPACE

```diff
@@ -13,6 +13,16 @@ local_repository(
     path = "envoy_build_config",
 )
 
+local_repository(
+    name = "envoy_mobile_bootstrap_config",
+    path = "envoy_mobile_bootstrap_config",
+)
+
+local_repository(
+    name = "extension_registry",
```

Comment:
right now I have created just a single extension registry, but my other idea was to perhaps have three extension registries:
1. extensions declared in envoy
2. extensions declared in envoy-mobile
3. extensions declared in envoy-mobile private

This way if a consumer is not interested in overriding 1, and 2, they don't need to.

Reply:
2. I think is a special case that can be omitted - the only reason envoy-mobile declare an extension is if its integral to the project, right?

1. and 3. seem like they can be combined.

Reply:
yep, for simplicity, I am going to leave just the one override point then.